src/frontend/Komponenter/Behandling/Vurdering/VurderingRedigeringsmodus.module.css

```diff
@@ -0,0 +1,7 @@
+.alert {
+    width: 25rem;
+}
+
+.aksjonsknapp {
+    width: fit-content;
+}
```

Comment:
Mulig man kan lage noen felles-moduler for mye brukte bredder etterhvert. Vi får se hvordan det blir etterhvert som flere komponenter tar i bruk moduler